docs/dev/reference/apis/services/vision.md

```diff
@@ -22,6 +22,10 @@ While the camera component lets you access what your machine's camera sees, the
 
 The vision service {{< glossary_tooltip term_id="api" text="API" >}} allows you to get detections, classifications, or point cloud objects, depending on the {{< glossary_tooltip term_id="ml" text="ML" >}} model the vision service is using.
 
+{{% alert title="Note" color="note" %}}
+The vision service can optionally depend on the [frame system](/operate/reference/services/frame-system/) for certain applications, particularly when working with 3D segmentation, spatial transformations, or when you need to relate camera coordinates to other components in your machine. For basic 2D detection and classification tasks, frame system configuration is typically not required.
+{{% /alert %}}
+
```

Comment:
```suggestion
```
This seems like an unnecessary box here; we don't need to overwhelm users with it at the top of the page I don't think.